pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -119,6 +119,7 @@ typedef struct __TypeContext {
 typedef struct __PyObjectEncoder {
     JSONObjectEncoder enc;
 
+
```

Comment:
minor nit but can you revert any added / removed blank lines in the PR? Should minimize the diff a bit more